### ISSUE TITLE
Crash report from Dynamo 2.12.1.8246

### ISSUE THREAD

**Issue by doughbear8** [DynamoDS/Dynamo#14192]
# Issue Description
Please fill in the following information to help us reproduce the issue: 

## Dynamo version
Dynamo: `2.12.1.8246`

## Operating system
OS: `Microsoft Windows NT 10.0.19045.0`

## What did you do?
Clicked an output on a Data-Shapes node to connect it to another node (I may have double-clicked)

## What did you expect to see?
A wire to start to connect to another node

## What did you see instead?
Autocomplete popped up, and Dynamo crashed

### Loaded Packages
- archi-lab.net 2022.212.3222
- bimorphNodes 3.0.3
- Clockwork for Dynamo 2.x 2.3.0
- Data-Shapes 2022.2.96
- LunchBox for Dynamo 2016.11.11
- Monocle 2022.9.1
- Rhythm 2023.6.2022
- spring nodes 204.1.0
- GenerativeDesign 1.10.10.0
- GenerativeDesign.Extension 1.10.10.0
- GenerativeDesign.Revit 1.10.10.0

## Stack Trace
```
(From the Dynamo crash window select 'Details' -> 'Copy' and paste here)
```

---
CLR: `4.0.30319.42000`

**Comment by QilongTang** [comment]
FYI: @avidit

**Comment by avidit** [comment]
Tracked internally [DYN-6100](https://jira.autodesk.com/browse/DYN-6100)

**Comment by mjkkirschner** [comment]
Hi @doughbear8 - is there a stack trace you can provide?

**Comment by doughbear8** [comment]
How do I go about getting a stack trace?

Thanks,
Beth

**Comment by avidit** [comment]
@mjkkirschner I have added steps to reproduce and attached stack trace in the jira issue.

**Comment by avidit** [comment]
Fixed in 2.19
